src/test/java/com/cdancy/bitbucket/rest/features/DefaultReviewersApiLiveTest.java

```diff
@@ -48,6 +53,67 @@ public void testListDefaultReviewersOnNewRepo() {
         assertThat(conditionList).isEmpty();
     }
 
+
+    @Test(dependsOnMethods = {"testListDefaultReviewersOnNewRepo"})
+    public void testCreateCondition() {
+        Long requiredApprover = 1L;
+        Matcher matcherSrc = Matcher.create(Matcher.MatcherId.ANY, true);
+        Matcher matcherDst = Matcher.create(Matcher.MatcherId.ANY, true);
+        List<User> listUser = new ArrayList<>();
+        listUser.add(getDefaultUser());
+        CreateCondition condition = CreateCondition.create(null, generatedTestContents.repository, matcherSrc,
+                matcherDst, listUser, requiredApprover);
+
+        Condition returnCondition = api().createCondition(projectKey, repoKey, condition);
+        conditionId = returnCondition.id();
+        validCondition(returnCondition, requiredApprover, Matcher.MatcherId.ANY_REF, Matcher.MatcherId.ANY_REF);
+    }
+
+    @Test(dependsOnMethods = {"testListDefaultReviewersOnNewRepo", "testCreateCondition"})
```

Comment:
No, I need to depend on method testCreateCondition and testCreateConditionMatcherDifferent because testListConditions do check if It have 2 conditions return by api. I will remove third.